api/tests/integration/audit/test_audit_logs.py

```diff
@@ -250,7 +250,7 @@ def test_retrieve_audit_log_includes_changes_when_segment_override_created_and_d
     get_audit_logs_response_2 = admin_client.get(get_audit_logs_url)
     assert get_audit_logs_response_2.status_code == status.HTTP_200_OK
     results = get_audit_logs_response_2.json()["results"]
-    assert len(results) == 5
+    assert len(results) == 6
```

Comment:
What's the audit log record that we've added here?

Reply:
I've fixed this back to the original value since our conversation about the versioned segments impact on the audit log